hw2/03-count.js

```diff
@@ -0,0 +1,20 @@
+// Add your code here
+const input = document.querySelector('input');
+const content = document.querySelector('#content');
+const main = document.querySelector('main');
+const result = document.createElement('div');
+main.appendChild(result);
+document.querySelector('body').style.backgroundColor = '#13415b';
+
+function handleKeyDown() {
+  const userInput = input.value;
+  const InitalTextHtml = content.innerHTML;
+  content.style.display = 'None'; // Doing this so I don't lose my original text
```

Comment:
instead of hiding the exisitng text area and making a new element, you could try saving the text content of the text area as a const, and use that text in your function to create a new block of text with the highlights. the text area can then be updated with the highlighted text

Reply:
Good point! I will make this change.